sirepo/package_data/static/js/radia.js

```diff
@@ -1343,6 +1349,144 @@ SIREPO.app.directive('bevelTable', function(appState, panelState, radiaService)
     };
 });
 
+SIREPO.app.directive('filletTable', function(appState, panelState, radiaService) {
+    return {
+        restrict: 'A',
+        scope: {
+            field: '=',
+            fieldName: '=',
+            itemClass: '@',
+            model: '=',
+            modelName: '=',
+            parentController: '=',
+            object: '=',
+        },
+        template: `
+            <table class="table table-hover">
+              <colgroup>
+                <col style="width: 20ex">
+                <col style="width: 20ex">
+                <col style="width: 20ex">
+              </colgroup>
+              <thead>
+                <tr>
+                  <th>Axis</th>
+                  <th>Edge</th>
+                  <th>Radius</th>
+                  <th>Resolution</th>
+                  <th></th>
+                </tr>
+              </thead>
+             <tbody>
+            <tr>
```

Comment:
Indentation seems off. Also applies to bevelTable

Reply:
[681e384](https://github.com/radiasoft/sirepo/pull/4699/commits/681e384af5430814034d4778780c0f6a0ff5ab39)